eraser-map.yaml

```diff
@@ -919,7 +919,7 @@ layers:
         data: { source: osm, layer: water }
         draw:
             grid:
-                order: 5
+                order: 2
```

Comment:
Good catch! Another reason to add more elbow room between the landuse and roads orders :)